src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -21,11 +21,28 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <PropertyGroup>
+      <_EmbeddedResourceSourceExtension Condition="'$(Language)' == 'C#'">cs</_EmbeddedResourceSourceExtension>
+      <_EmbeddedResourceSourceExtension Condition="'$(Language)' == 'VB'">vb</_EmbeddedResourceSourceExtension>
+  </PropertyGroup>
 
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' ">
     <Compile Include="**/*$(DefaultLanguageSourceExtension)" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" Condition=" '$(EnableDefaultCompileItems)' == 'true' " />
     <EmbeddedResource Include="**/*.resx" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' " />
+
+    <EmbeddedResource Update="**\*.resx" Condition="'$(_EmbeddedResourceSourceExtension)' != ''">
+      <LastGenOutput Condition= "%(LastGenOutput) == ''">$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileName(%(Identity))), '.Designer.$(_EmbeddedResourceSourceExtension)'))</LastGenOutput>
+      <Generator Condition= "%(Generator) == ''">ResXFileCodeGenerator</Generator>
+    </EmbeddedResource>
+
+    <Compile Update="**\*.Designer.$(_EmbeddedResourceSourceExtension)" Condition="'$(_EmbeddedResourceSourceExtension)' != ''">
```

Comment:
⚠️ There are many other generated source files which match this pattern, including Windows Forms.

Reply:
@sharwell good point. And I just talked to Daniel, that sounds like a hard problem. I will set up a meeting for it